src/common/services/Topic/getLearningPathTopic.tsx

```diff
@@ -0,0 +1,29 @@
+import { LearningPathTopic, LearningPathTopicReturn } from '@core'
+
+export const getLearningPathTopic: LearningPathTopicReturn = async (userId, lmsUserId, studentId, course_id) => {
+  if (!course_id) {
+    throw new Error('course_id is required')
+  }
+  return fetch(process.env.BACKEND + `/user/${userId}/${lmsUserId}/student/${studentId}/course/${course_id}/topic`, {
+    method: 'GET',
+    credentials: 'include',
+    headers: {
+      'Content-Type': 'application/json'
+    }
+  }).then((response) => {
```

Comment:
Hier vllt auch. Evtl kleine Anpassungen am test notwendig.
```suggestion
  const response ...
  getData<T>(response)
```